src/okta_inspector/reporters/irap.py

```
@@ -0,0 +1,365 @@
+        pw_pass = any(
+            f.status == "pass"
+            for cid, f in irap_findings.items()
+            if cid in ("ISM-0421", "ISM-1593")
+        )
+
+        lines: list[str] = [
+            "# Essential Eight Maturity Assessment",
+            "",
+            f"**Audit Timestamp:** {ts}  ",
+            f"**Okta Domain:** {data.domain}  ",
+            f"**Assessment Basis:** ASD Essential Eight Maturity Model",
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)